build.gradle.kts

```diff
@@ -184,6 +184,14 @@ dependencies {
     implementation("org.apache.cxf:cxf-rt-features-logging:${Versions.cxfVersion}")
     implementation("org.apache.cxf:cxf-rt-transports-http:${Versions.cxfVersion}")
     implementation("org.apache.cxf:cxf-rt-ws-security:${Versions.cxfVersion}")
+    constraints {
+        implementation("org.apache.santuario:xmlsec") {
+            because("org.apache.cxf:cxf-rt-ws-security:${Versions.cxfVersion} -> https://nvd.nist.gov/vuln/detail/CVE-2023-44483")
+            version {
+                require("2.3.4")
+            }
+        }
+    }
```

Comment:
Vil dette gjøre at avhengigheten alltid er `2.3.4`? 😊 Ser at versjon `3.0.3` også er safe. Burde vi vært på denne? Ser ut som nyeste er `4.0.3` burde vi bare oppgradert til denne? 🤔

Reply:
Jeg tenkte det er greit å ikke flytte seg for langt unna det som "egentlig" brukes, fordi det er større sjanse for at noe brekker. Men det er ikke sikkert det er et problem 🤔

Reply:
Godt poeng! Kjør `2.3.4`🚀